cypress/support/fragments/users/loans/userLoans.js

```diff
@@ -52,12 +79,30 @@ export default {
     cy.do(itemDetailsButton.click());
     ItemVeiw.waitLoading();
   },
-  renewItem:(barcode) => {
-    cy.expect(rowInList.find(HTML(including(barcode))).exists());
-    cy.do(ellipsisButton.click());
-    cy.expect(renewButton.exists());
-    cy.do(renewButton.click());
+  renewItem:(barcode, isLoanOpened = false) => {
+    if (isLoanOpened) {
+      cy.expect(KeyValue({ value: barcode }).exists());
+      cy.expect(renewButton.exists());
+      cy.do(renewButton.click());
+    } else {
+      cy.expect(rowInList.find(HTML(including(barcode))).exists());
+      cy.do(ellipsisButton.click());
+      cy.expect(renewButton.exists());
+      cy.do(renewButton.click());
+    }
   },
+  // reneweThroughOverride: (mes) => {
+  //   cy.expect([
+  //     Modal('Renew Confirmation').exists(),
+  //     overrideButton.exists()
+  //   ]);
+  //   cy.do([
+  //     overrideButton.click(),
+  //     TextArea({ id: 'data-test-additional-info' }).fillIn(mes),
+  //     CheckboxInTable({ name: 'check-all' }).click(),
+  //     Modal({ id:'bulk-override-modal' }).find(overrideButton).click()
+  //   ]);
+  // },
```

Comment:
do you need this commented code in this pull request?

Reply:
Removed